app/models/task.rb

```diff
@@ -15,8 +15,10 @@
 #  provider_job_id        :string
 #  updated_at             :datetime         not null
 #
-
 class Task < ApplicationRecord
+  scope :full_fetcher, -> { where(job_class: 'FullFetcherJob') }
```

Comment:
Should we use `job_type_value` for these?

Reply:
use :full_fetcher, and make a constant TASK_JOB_TYPES in the Task model file. Validation in Task that task_job_type is in that constant